hnn_core/dipole.py

```diff
@@ -448,6 +448,10 @@ def plot_psd(self, fmin=0, fmax=None, tmin=None, tmax=None, layer='agg',
             simulation.
         layer : str, default 'agg'
             The layer to plot. Can be one of 'agg', 'L2', and 'L5'
+        color : str or tuple or None
+            The line color of PSD
+        label : str or None
+            Line label for PSD
```

Comment:
Just to maintain style consistency...
```suggestion
        color : str | tuple | None
            The line color of PSD
        label : str | None
            Line label for PSD
```